packages/frontend/amp/components/Onward.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { css } from 'react-emotion';
+import Plus from '@guardian/pasteup/icons/plus.svg';
+import InnerContainer from '@frontend/amp/components/InnerContainer';
+import { OnwardContainer } from '@frontend/amp/components/OnwardContainer';
```

Comment:
Some modules have default exports and some have named exports. I think we agreed to standardise on [named exports](https://github.com/guardian/dotcom-rendering/blob/master/docs/contributing/code-style.md#always-used-named-exports), does that work for you?